.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: node_js
 
 node_js:
+  - iojs
+  - '0.12'
```

Comment:
@DanielRosenwasser I just think it is common practice to test multiple environments. Some test fails sometimes on different environments but not others.